### PR TITLE
Amazon SageMaker Endpoint Driver

### PR DESCRIPTION
**Pull request by shinglyu**
This driver allows you to invoke LLMs running on Amazon SageMaker Endpoint, including SageMaker JumpStart models. See the README for a getting started guide.

### CODE REVIEW

**Comment by williamhogman** [comment]
Great work so far. Appericiate you taking the time :)

**Comment by shinglyu** [comment]
I ran cargo fmt on all the crates that fails the CI/CD. There is an error on [llm-chain-milvus](https://github.com/sobelio/llm-chain/tree/main/crates/llm-chain-milvus) being unlicense, which I don't know how to fix.

**Comment by williamhogman** [comment]
Thanks will take a look

File: crates/llm-chain-sagemaker-endpoint/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# llm-chain-mock
```
Comment (williamhogman) [inline crates/llm-chain-sagemaker-endpoint/README.md:1]: Looks like the wrong title :)

**Review by williamhogman — COMMENTED** [review]
Just the title in the readme otherwise looks good to me

File: crates/llm-chain-sagemaker-endpoint/README.md (hunk 1 above)
Comment (shinglyu) [reply crates/llm-chain-sagemaker-endpoint/README.md]: Fixed!

**Pull request merged**